osu.Game.Rulesets.Osu/Edit/OsuToolboxComposite.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics.Containers;
+using osu.Game.Graphics.UserInterface;
+using osu.Game.Overlays.Settings.Sections;
+using osu.Game.Rulesets.Edit;
+using osu.Game.Screens.Edit;
+using osuTK;
+
+namespace osu.Game.Rulesets.Osu.Edit
+{
+    /// <summary>
+    /// A toolbox composite for osu!-specific controls.
+    /// </summary>
+    // todo: once catch supports distance spacing, the control here should move out to a base "DistancingRulesetToolboxComposite" class or something better.
```

Comment:
Strongly disagree with this todo. I'd much rather have this composite duplicated in catch than extract enterprise-grade inflexible base classes.

Reply:
I don't know about that, I could look at splitting the slider itself to avoid a bit of duplication, but the distance spacing scroll logic can't be shared without sharing this class itself. And basically doing the same for custom rulesets as well.

Reply:
See https://github.com/ppy/osu/pull/16576#discussion_r860370075, `OsuToolboxComposite` has just been removed in favour of `DistancedHitObjectComposer` which contains the distance spacing logic in there.